app/services/airports/getAirportByIataCode.ts

```
@@ -0,0 +1,14 @@
+import airports from "data/airports/data.json";
+import type { Airport } from "./type";
+
+export function getAirportByIataCode(
+  airportIataCode: string,
+): Airport | undefined {
+  if (airportIataCode.length === 0) {
+    return undefined;
+  }
```

if missing IATA code, we should throw or who handles this undefined?

---

I would prefer undefined, otherwise I need to handle the exception for every function how uses this function